pkgs/data/fonts/iosevka/update.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p curl jq
+
+set -e
+
+release=$(curl -s https://api.github.com/repos/be5invis/Iosevka/releases/latest)
+
+oldVersion=$(nix-instantiate --eval -E 'with import ./. {}; lib.getVersion iosevka-bin' | tr -d '"')
+version=$(echo "$release" | jq -r .tag_name | tr -d v)
+
+if test "$oldVersion" = "$version"; then
+	echo "New version same as old version, nothing to do." >&2
+	exit 0
+fi
+
+file=$(nix-instantiate --eval -A iosevka-bin.meta.position | sed -r 's/^"(.*):[0-9]+"$/\1/')
+sed -i "s/$oldVersion/$version/" "$file"
```

Comment:
could this use `update-source-version`?

Reply:
No, because it tries to update the hash, which doesn't exist (since I'm maintaining a list of hashes).

Reply:
I actually started with `update-source-version` and did this manually after I realised there was no way to disable this behaviour.

Reply:
update-source-version can take four arguments, attribute path, version, hash, url. I believe if you just run with three arguments it should do what you want (I've used this program extensively)

Reply:
I don't understand how? [this check](https://github.com/NixOS/nixpkgs/blob/d3025ee8c62cba6e92de8d3900beb73889ca01ca/pkgs/common-updater/scripts/update-source-version#L102-L104) straight up precludes me from running it on `bin.nix`, since it's looking for the hash of the default expression (i.e. no variants), and the hash isn't present in `bin.nix` at all. It also only supports updating a single hash, whereas there are many hashes in `variants.nix.` Moreover, variants come and go, so the updater script grabs the list dynamically.

Reply:
Oh sorry, I misread this as ~set $oldHash/$nevHash`. Plz ignore